airflow/operators/python_operator.py

```diff
@@ -334,7 +334,7 @@ def _execute_in_subprocess(self, cmd):
             self.log.info("Executing cmd\n%s", cmd)
             output = subprocess.check_output(cmd,
                                              stderr=subprocess.STDOUT,
-                                             close_fds=True)
+                                             close_fds=True).decode('utf-8')
```

Comment:
From: https://docs.python.org/3/library/subprocess.html#subprocess.check_output
> By default, this function will return the data as encoded bytes. The actual encoding of the output data may depend on the command being invoked, so the decoding to text will often need to be handled at the application level.

I think it is fine to decode it at that level, but then we should set the encoding in the `check_output` as well to `encoding='utf-8'`, don't you think so?

Reply:
I think it would be better to add an optional/kwarg to the `__init__` to pass over the command-specific encoding/decoding.